examples/rustfmt/module/rust/Cargo.toml

```diff
@@ -12,7 +12,9 @@ crate-type = ["cdylib"]
 log = "*"
 oak = "=0.1.0"
 prost = "*"
-rustfmt-nightly = "*"
+rustfmt-nightly = "=1.4.14"
+rustc-ap-rustc_session = "=654.0.0"
```

Comment:
Without these, Cargo uses version `659.0.0` and fails with:
```
error[E0407]: method `forward_checked` is not a member of trait `std::iter::Step`
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_span-659.0.0/def_id.rs:11:1
   |
11 | / rustc_index::newtype_index! {
12 | |     pub struct CrateId {
13 | |         ENCODABLE = custom
14 | |     }
15 | | }
   | |_^ not a member of trait `std::iter::Step`
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```
Is there a better way to fix this? I tried the newest nightly Rust, but even that did not fix the problem IIRC.

Reply:
Personally, I think we should just delete the `rustfmt` example; it doesn't illustrate anything that's not covered in other examples, it keeps on causing these sort of problems (cf. #588, #594), it it's one of the main things preventing us from moving to the Rust stable channel (#969)

Reply:
As the original author of that example, I'm fine to get rid of it, especially if it gets in the way of other things. (it is also quite slow to compile) @daviddrysdale do you want to do the honours?